boilerplate/app/i18n/en.json

```diff
@@ -10,7 +10,12 @@
   "welcomeScreen": {
     "poweredBy": "POWERED BY IGNITE",
     "readyForLaunch": "Ready for launch.",
-    "continue": "CONTINUE"
+    "continue": "CONTINUE",
+    "more": {
+      "things": {
+        "here": "Here"
+      }
+    }
```

Comment:
do you still need these?

Reply:
That was a TypeScript test, good catch!

![good catch](https://c.tenor.com/oPf4WuXQ2EIAAAAd/sports-football.gif)